_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+
+1. Choose **Dashboard** from the OpenSearch Dashboards main menu.
+2. From the **Dashboards** window, select **Create** and then choose **Dashboard**.
+3. Select **Add an existing**, then select the appropriate alerting visualization from the **Add panels** list. The visualization is added to the dashboard.
+4. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}). 
+5. From the **Options** menu, select **Add alerting monitor**.
+6. Input information for **Monitor details** and **Triggers**  
+7. Choose **Create monitor**. The monitor is added to the visualization.  
+
+An example of the create monitor steps is shown in the following screenshot.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/create-monitor-menu.png" alt="Create monitor interface" width="400" height="400">
+
+## Associating monitors
+
+You can associate certain monitors with a visualization using the Dashboard interface instead of the plugin page, giving you a single interface to add, view, and edit monitor data. You can associate existing monitors and monitors not specific to a per query monitor.
+
+#### To associate a monitor
+
+Continuing with the alerting visualization and dashboard created in the preceding tutorial, associate an existing monitor with a visualization by following these steps. 
+
+1. From the visualization panel, choose the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}).
+2. Select **Associated monitors**.
+3. From the **Select monitor to associate** dropdown menu, select the monitor. Only eligible monitors are listed in the dropdown menu. 
+4. View the monitor's basic information. To view comprehensive details, select **View monitor page** to open the Alerting plugin page.
+5. Select **Associate monitor**. An existing monitor is associated to the visualization.
+
+## Exploring alerting monitor details
+
+Once you've created or associated alerting monitors, verify the monitor is generating the alerts and explore alert details by following these steps:
+
+1. Open the alerting dashboard. Alerts are indicated on the visualization with a triangle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/dashboards/triangle-icon.png" class="inline-icon" alt="triangle icon"/>{:/}). 
+2. Hover over a triangle to view high-level data, such as number of alerts. To investigate alert details, select the triangle icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/dashboards/triangle-icon.png" class="inline-icon" alt="triangle icon"/>{:/}) to open a flyout with more detailed monitor information. Alternatively, select the ellipsis icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/ellipsis-icon.png" class="inline-icon" alt="ellipsis icon"/>{:/}) in the visualization panel and choose **View events**.
```

Comment:
"Hover over a triangle to view high-level data, such as number of alerts."
Specify that we're hovering over a "triangle icon" to avoid the possibility a reader thinks there might be another kind of three-sided geometric shape to look for?